boards/v2/qemu/qemu_cortex_a9/Kconfig.qemu_cortex_a9

```diff
@@ -6,6 +6,5 @@
 #
 
 config BOARD_QEMU_CORTEX_A9
-	bool "Cortex-A9 Emulation (QEMU)"
-	depends on SOC_XILINX_XC7Z007S
+	select SOC_XILINX_XC7Z007S
 	select QEMU_TARGET
```

Comment:
the `select QEMU_TARGET` must either go to the `<board-dir>/Kconfig` or we must have a config qemu target in the Kconfig.soc tree.

Reply:
Fixed